README.md

```diff
@@ -7,20 +7,67 @@ Named after [the artist](https://artsy.net/artist/kaws), kaws is a backend servi
 * __State:__ in development
 * __Production:__ [k8s](https://kubernetes.artsy.net/#!/deployment/default/kaws-web?namespace=default)
 * __Staging:__  [http://kaws-staging.artsy.net](http://kaws-staging.artsy.net/playground) | [k8s](https://kubernetes-staging.artsy.net/#!/search?q=kaws&namespace=default)
-* __Github:__ [https://github.com/artsy/positron/](https://github.com/artsy/kaws/)
+* __Github:__ [https://github.com/artsy/kaws/](https://github.com/artsy/kaws/)
 * __CI:__ [CircleCI](https://circleci.com/gh/artsy/kaws); merged PRs to artsy/kaws#master are automatically deployed to staging. PRs from `staging` to `release` are automatically deployed to production. [Start a deploy...](https://github.com/artsy/kaws/compare/release...staging?expand=1)
 * **Point People:** [@l2succes](https://github.com/l2succes)
 
-## How do I work on this?
+## Prerequisites
 
-```sh
-git clone https://github.com/artsy/kaws.git
-cd kaws
-yarn install
+ * node.js 8.12.0 or newer
+ * yarn 1.10.1 or newer
+ * MongoDB 4.x
```

Comment:
I just picked up the latest versions but let me know if we are targeting something else.

Reply:
Yup that looks good